src/System.Diagnostics.Process/tests/ProcessCollectionTests.cs

```diff
@@ -12,7 +12,7 @@ public class ProcessCollectionTests : ProcessTestBase
         [Fact]
         public void TestModuleCollectionBehavior()
         {
-            ProcessModule[] mArray = _process.Modules.Cast<ProcessModule>().ToArray();
+            ProcessModule[] mArray = Process.GetCurrentProcess().Modules.Cast<ProcessModule>().ToArray();
```

Comment:
```_process.Start``` is starting ```RemoteExecutorConsoleApp.exe```, which is built for ```Any CPU```, and ```Prefer 32bit``` property is not set on the project. So while running on 64 bit machines, it will start as 64 bit process, and the ```xunit.console.netcore.exe``` is a 32bit process. ```Process.GetCurrentProcess()``` will access the 32 bit ```xunit.console.netcore.exe``` process so there should be no bitness problem.